plugins/inputs/lotus/lotus.go

```diff
@@ -464,6 +469,47 @@ func recordTipsetMessagesPoints(ctx context.Context, api api.FullNode, acc teleg
 	return nil
 }
 
+func recordLotusInfoPoints(ctx context.Context, api api.FullNode, acc telegraf.Accumulator) error {
+	nodePeerID, err := api.ID(context.Background())
+	if err != nil {
+		return err
+	}
+
+	v, err := api.Version(ctx)
+	if err != nil {
+		return err
+	}
+
+	network, err := api.StateNetworkName(ctx)
+	if err != nil {
+		return err
+	}
+	var commit string
+	var version string
+	versionTokens := strings.Split(v.Version, "+")
+	if len(versionTokens) == 2 {
+		version = versionTokens[0]
+		commit = versionTokens[1]
+	} else {
+		log.Println("W! developer error, version string has changed format")
+		version = v.Version
+	}
+
+	acc.AddFields("lotus_info",
+		map[string]interface{}{
+			"recorded_at": time.Now().UnixNano(),
```

Comment:
Sorry I overlooked this the first time. A timestamp is automatically associated with each `AddFields` call. This fields will double the data we capture for little extra value.